abr_control/utils/target.py

```diff
@@ -0,0 +1,145 @@
+"""
+"""
+import numpy as np
+import random
+
+class Target():
```

Comment:
target generation class, does it make sense in abr_control?

Reply:
nope! dewolf2018!